spec/controllers/sessions_controller_spec.rb

```diff
@@ -0,0 +1,75 @@
+require "rails_helper"
+
+describe SessionsController do
+  describe 'GET new' do
+    it "renders new template for unauthenticated users" do
+      get :new
+      expect(response).to render_template :new
```

Comment:
if user is not authenticated this should be default behaviour of Rails, so you should not test it. `GET new` -> `renders :new`